plugin/pkg/scheduler/metrics/metrics.go

```diff
@@ -68,3 +80,17 @@ func Register() {
 func SinceInMicroseconds(start time.Time) float64 {
 	return float64(time.Since(start).Nanoseconds() / time.Microsecond.Nanoseconds())
 }
+
+// ReportBindingRateLimiterSaturation exports status of binding rate limiter
+// whether it's full (1) or not (0).
+func ReportBindingRateLimiterSaturation(rl util.RateLimiter) {
```

Comment:
This is not binding specific code, so remove that from the name (and pass the metric to set in as a parameter).